packages/repository-modernizer/src/util/constants.js

```diff
@@ -65,9 +65,9 @@ module.exports = {
 
     NON_ADOBE_DEPENDENCIES: "nonadobedependencies",
 
-    BUNDLE: "bundle",
+    BUNDLE: ["bundle"],
```

Comment:
Why do you need to change this? I don't it is being used for checking the packaging-type. Also this is expected to be of string type (used in log msg [here](https://github.com/adobe/aem-cloud-service-source-migration/blob/master/packages/repository-modernizer/src/create-base-project-structure.js#L411)).